tests/test_lockfree_set.cpp

```diff
@@ -364,12 +364,12 @@ Test_Set lockfree_set_tests()
 {
     return
     {
-        { "empty", empty },
-        { "contains", contains },
-        { "iteration", iteration },
+        { "empty", test_empty },
```

Comment:
Test names needed to be changed when raising the C++ standard level - the `empty` symbol clashed with parts of standard library, causing a misleading error.